public/app/features/dashboard-scene/pages/DashboardScenePage.tsx

```diff
@@ -23,6 +23,7 @@ export interface Props
 export function DashboardScenePage({ route, queryParams, location }: Props) {
   const params = useParams();
   const { type, slug, uid } = params;
+  const path = params['*'];
```

Comment:
The `path` can contain slashes, so we need to use the wildcard param to get it.

Reply:
Should we leave a comment saying this is used by:
```
/admin/provisioning/:slug/dashboard/preview/*
```
to load dashboards based on their file path in a remote repository.  or similar?